src/components/SelQuestCont/SelQuestCont.js

```diff
@@ -1,9 +1,17 @@
 import './SelQuestCont.css';
+import { getSelQuestion } from '../../apiCalls';
+import { useState } from 'react';
+import { useParams } from 'react-router-dom';
 
 function SelQuestCont() {
```

Comment:
Is there a better variable name to use here? As someone who is unfamiliar with the code base, I'm not sure what this component is doing!